docs/src/main/sphinx/sql/create-catalog.md

```diff
@@ -0,0 +1,44 @@
+# CREATE CATALOG
+
+## Synopsis
+
+```text
+CREATE CATALOG
+catalog_name
+USING connector_name
+[ WITH ( property_name = expression [, ...] ) ]
+```
+
+## Description
+
+Create a new catalog using the specified connector. The catalog is ready to be
+used as soon as this query completes.
+
+The optional `WITH` clause can be used to set properties
+on the newly created catalog. Property names can be double quoted, which is
+required if they contain special characters, like `-`.
```

Comment:
Add something about valid properties depend on the connector .. refer to the docs for those.

Also how does it behave if you create invalid config? Is there any validation or does it just create them and then you have to test if it actually worked afterwards.

Reply:
Added all of this and a long list of how properties are validated, thanks!